test/scaling/withextrap.jl

```diff
@@ -0,0 +1,40 @@
+
+module ScalingWithExtrapTests
```

Comment:
I love how composable this framework is, and that you just get this "for free" :smile:.